docs/develop/howtos/override_components.md

```diff
@@ -0,0 +1,121 @@
+# How to override a react component
+
+This documentation is targeted to developers that want to customize a specific react component in an instance.
+
+## Overridable
+
+It is possible to override certain react components in Invenio RDM, by using the library [`react-overridable`](https://github.com/indico/react-overridable). This feature is available to instance developers to customize the look and feel of the react application (e.g. deposit form).
```

Comment:
I would add a very short explanation of what you can do with `react-overridable` e.g. override React components by unique cmp ids.